### PR TITLE
[Exporter.Instana] Bump OTel to 1.4.0

### PR DESCRIPTION
**Pull request by Kielek**
Follow up to #1038 where Instana was unpinned from $(OpenTelemetryCoreLatestVersion)
Towards #1031

## Changes

Bump OTel 1.4.0, as this version is not supporting .NET Framework 4.6.1 and .NET Core App 3.1
The package will produce libraries for .NET Framework 4.6.2 and test was switched to `net7.0;net.60;net462`

For significant contributions please make sure you have completed the following items:

* [x] Appropriate `CHANGELOG.md` updated for non-trivial changes
* ~~[ ] Design discussion issue #~~
* ~~[ ] Changes in public API reviewed~~

### CODE REVIEW

**Comment by Kielek** [comment]
@zivaninstana, could you please review?

**Review by zivaninstana — APPROVED** [review]
LGTM 👍

File: test/OpenTelemetry.Exporter.Instana.Tests/OpenTelemetry.Exporter.Instana.Tests.csproj [hunk 1]
```diff
@@ -13,6 +14,7 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.2" Condition="'$(TargetFramework)' == 'net462'"/>
```
Comment (utpilla) [inline test/OpenTelemetry.Exporter.Instana.Tests/OpenTelemetry.Exporter.Instana.Tests.csproj:17]: Why is this being added?

Comment (Kielek) [reply test/OpenTelemetry.Exporter.Instana.Tests/OpenTelemetry.Exporter.Instana.Tests.csproj:17]: Microsoft.NET.Test.Sdk -> [Microsoft.TestPlatform.TestHost](https://www.nuget.org/packages/Microsoft.TestPlatform.TestHost/17.3.2#dependencies-body-tab) -> Newtonsoft.Json for .NET (Core). There is no such reference for .NET Framework.

At least one tests needs it https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/e4582ffece4d43254f34d081aec78dfea83073ac/test/OpenTelemetry.Exporter.Instana.Tests/InstanaSpanSerializerTests.cs.

As I added execution tests for NET Fx 4.6.2 in this PR this change is needed.

**Review by cijothomas — APPROVED** [review]

**Pull request merged**